### PR TITLE
test(ovm): pass test

### PR DESCRIPTION
**Pull request by syuhei176**

### CODE REVIEW

File: packages/db/src/getWitnesses.ts [hunk 1]
```diff
@@ -55,7 +55,11 @@ export async function getWitnesses(
     }
     return result
   } else if (type === 'NUMBER') {
-    const [start, end] = param.split('-').map(n => BigNumber.fromHexString(n))
+    const [start, end] = param
+      .split('-')
+      .map(n =>
+        ovmContext.coder.decode(BigNumber.default(), Bytes.fromHexString(n))
```
Comment (syuhei176) [inline packages/db/src/getWitnesses.ts]: getWitness depends on coder but db module should not. consider to move getWitness.

Comment (tkmct) [reply packages/db/src/getWitnesses.ts]: I think it's fine!

File: packages/db/package.json [hunk 2]
```diff
@@ -48,6 +48,8 @@
     }
   },
   "dependencies": {
+    "@cryptoeconomicslab/coder": "0.0.6",
```
Comment (tkmct) [inline packages/db/package.json]: can you put `^` for version?

Comment (syuhei176) [reply packages/db/package.json]: fixed

**Pull request merged**